diagnostic.md

```diff
@@ -32,6 +49,14 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
+Getter:
+$('selector').find()
+$('selector').text()
+
+Setter:
+$('selector').append()
+$('selector').prepend()
+
```

Comment:
Good, also the jQuery getter methods become setters when you pass them an argument to use as the value.